locale/fi/LC_MESSAGES/django.po

```diff
@@ -6,7 +6,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: \n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2020-06-24 15:50+0300\n"
+"POT-Creation-Date: 2020-07-03 10:03+0300\n"
 "PO-Revision-Date: 2019-08-14 12:16+0200\n"
 "Last-Translator: Saara Satokangas <saara.satokangas@aalto.fi>\n"
```

Comment:
To my understanding, it's usually unnecessary to update/commit these (as we use git) and their updation easily causes merge-conflicts

Reply:
You mean the POT-Creation-Date changes or the translation changes? 

I like to add the changes in translations commit that change them, as I think that the creation-date and who made the changes to translations is important if the state of the branch would be reverted to some specific point (commit) in time. And I also like to ensure that they work before I commit/create a pull request. I guess they could be separated into their own commit at the end of a PR. Though then the branch would not necessarily work after each commit ... hmm.. not sure what is the best way. 

But for example, with my previous pull request, some new translations appeared from other earlier pull requests, since those files had not been committed after their final changes in the related PRs. It looked a bit weird as I hadn't wrote those translations, and at least I try to prevent that from happening.

Reply:
I mean the POT-Creation-Date and Last-Translator changes. The translations themselves should definitely be part of the same commit as the things that are changed.
I agree that it's kind of weird that there are new changes that aren't mentioned in the meta-data (and I think it's even more weird that the revision date is before the creation date), but when I've asked @raphendyr about whether to contain those POT-Creation-Dates and Last-Translator changes, he's always instructed to leave them out. He's probably able to give more input on it.

Reply:
I asked him whether to include them or not: "saa, ei oo pakko. pitää keksiä style guideen jotain"

Reply:
Alrighty, that clears it up, thanks :)